packages/tenant-process/src/services/tenantService.ts

```diff
@@ -482,4 +547,28 @@ export function tenantServiceBuilder(
     },
   };
 }
+
+export function reAssignAttribute({
```

Comment:
```suggestion
function reAssignAttribute({
```
I don't think we'll need this outside this scope